eager_robot_ur5e/config/ur5e.yaml

```diff
@@ -109,13 +109,12 @@ gazebo:
       - wrist_2_joint
       - wrist_3_joint
 
-  states: # IMPORTANT! Only add states you are able to both read and reset in the physics engine
+  # IMPORTANT! Currently, not possible to mix states consisting of both links & joints,
+  # IMPORTANT! Only add states you are able to both read and reset in the physics engine (base_pose, base_vel, joint_pos, joint_vel)
```

Comment:
Well, link resets are not yet implemented, but it would get problematic, because if a link and its revolute joint are reset, either one of the resets will be overwritten by the other right?

Reply:
And I thought that the second comment holds for all engines, or did I get that wrong?